content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -227,7 +227,7 @@ have some limitations:
 
 - Most of the metadata about a Pod is immutable. For example, you cannot
   change the `namespace`, `name`, `uid`, or `creationTimestamp` fields;
-  the `generation` field is unique. It only accepts updates that increment the
+  the `generation` field is unique. It only accepts updates that increase the
```

Comment:
@minhtaile2712, an increase doesn't happen at a fixed amount, unlike an increment. You can try the demo below:

**deployment.yaml**
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-increment
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:alpine
        ports:
        - containerPort: 80
```
```bash
kubectl apply -f deployment
sleep 5
kubectl get deployments -o yaml | grep -i generation
```
You'll get the below output:
```bash
    generation: 1
    observedGeneration: 1
```

Now update the image tag and view the `generation` value:
```bash
kubectl set image deployment/cluster-exec-deployment nginx=nginx:1.20
kubectl get deployments -o yaml | grep -i generation
```

Notice the increment in the below output:
```bash
    generation: 2
    observedGeneration: 2
```
Also, according to this [dictionary](https://en.bab.la/dictionary/english/increment), an increment can be a verb in a computing context.